src/CodeGen_Vulkan_Dev.cpp

```diff
@@ -0,0 +1,2924 @@
+#include <algorithm>
+#include <fstream>  // for dump to file
+#include <sstream>
+#include <unordered_set>
+
+#include "CSE.h"
+#include "CodeGen_GPU_Dev.h"
+#include "CodeGen_Internal.h"
+#include "CodeGen_Vulkan_Dev.h"
+#include "Debug.h"
+#include "Deinterleave.h"
+#include "FindIntrinsics.h"
+#include "IROperator.h"
+#include "IRPrinter.h"
+#include "Scope.h"
+#include "Simplify.h"
+#include "SpirvIR.h"
+#include "Target.h"
+
+#ifdef WITH_SPIRV
+
+namespace Halide {
+namespace Internal {
+
+namespace {  // anonymous
+
+// --
+
+template<typename CodeGenT, typename ValueT>
+ValueT lower_int_uint_div(CodeGenT *cg, Expr a, Expr b);
+
+template<typename CodeGenT, typename ValueT>
+ValueT lower_int_uint_mod(CodeGenT *cg, Expr a, Expr b);
```

Comment:
Are these ever used or defined? I don't see them anywhere else.

Reply:
Ah, good catch.  These declarations inside the anonymous namespace are no longer used ... the real definitions are inside the Internal namespace and declared in CodeGen_Internal.  I'll remove these! Thx!